src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const headers = document.querySelectorAll('th');
+const tBody = document.querySelector('tBody');
+const rows = document.querySelectorAll('tBody tr');
+
+const arrayOf = [...rows];
+
+headers.forEach((header) => {
+  header.addEventListener('click', function () {
```

Comment:
You are using a regular function here. Consider using an arrow function for consistency with the rest of your code.